packages/account/src/Sections/index.js

```diff
@@ -9,7 +9,7 @@ import ProofOfOwnership from 'Sections/Verification/ProofOfOwnership';
 import TwoFactorAuthentication from 'Sections/Security/TwoFactorAuthentication';
```

Comment:
kindly migrate index to ts 🚚

Reply:
I am getting `Parse errors in imported module 'Sections/Verification/ProofOfAddress': ';' expected.` I am guessing it wants the imported and exported to be tsx?

Reply:
in the respective index file of `ProofOfAdress`, can you write
`export { default } from ****`
instead of 
`export default from ****`
will this work?